### PR TITLE
[docs] Add template configuration files

### PR DESCRIPTION
**Pull request by VSevagen**
This commit contains the configuration files
for the template we'll use for the tutorial
and an updated README on how to built it.

Signed-off-by: sevagenv <sevagenv@gmail.com>

### CODE REVIEW

**Comment by GeorgLink** [comment]
woot woot -- I'm super excited that the new tutorial is moving into this repository 🎉🎉🎉🎉

File: README.md [hunk 1]
```diff
@@ -1,82 +1,28 @@
...
-GrimoireLab supports more than 20 different kinds of [data sources](https://gitlab.com/Bitergia/c/FAQ/tree/master/sources#supported-data-sources), from git repositories or GitHub projects, to issue trackers such as Jira or Bugzilla, including messaging systems such as IRC, Slack or mailing lists, or other types of systems such as StackOverflow or Jenkins. 
+# How to set it up locally
 
-With the tools in the GrimoireLab toolset you can retrieve data, store it in databases, analyze it, and produce nice dashboards for interactively visualizing the resulting information.
+Clone this repository
 
-![](eclipse.png)
-
-Example dashboard produced with GrimoireLab: the [CHAOSS Development Dashboard](https://chaoss.biterg.io)
-
```
Comment (vchrombie) [inline README.md:14]: I would mark this comment as marking.

I would revert the changes above this marking and move the below content to the `index.md` file. The below content i.e, quick overview of the grimoirelab tools would look good on the first page of the tutorial.

File: README.md [hunk 2]
```diff
@@ -1,82 +1,28 @@
...
 
+Then [start serving the site locally](https://help.github.com/en/github/working-with-github-pages/testing-your-github-pages-site-locally-with-jekyll):
 
-![](assets/Chaoss_Logo_Pantone1.png)
+```bash
+$ bundle exec jekyll serve
+```
 
-GrimoireLab is a [CHAOSS](http://chaoss.community) project.
+Open the site in the browser of your choice: http://127.0.0.1:4000
```
Comment (vchrombie) [inline README.md:28]: These installation steps should better go to CONTRIBUTING.md file. WDYT?

**Review by vchrombie — CHANGES_REQUESTED** [review]
Thanks for opening the PR @VSevagen.

I have the first set of comments here.

Also, do you think we need to update the `.gitignore` file?

**Comment by VSevagen** [comment]
> Also, do you think we need to update the `.gitignore` file?

I think the current one is fine. Using the doc locally or even after deploying does not create any additional data other than `_site`. I have the same content for my .gitignore file in the new tutorial.

File: README.md (hunk 2 above)
Comment (VSevagen) [reply README.md:28]: Yes. We can add them. We already have a section that redirects to CONTRIBUTING.md so we can update the latter accordingly

Comment (VSevagen) [reply README.md]: I have a small doubt. 
The installation steps should be added [here](https://github.com/chaoss/grimoirelab-tutorial/blob/master/contributing.md) right or did you mean [here](https://github.com/chaoss/grimoirelab/blob/master/CONTRIBUTING.md) since you mentioned contributing.md as CONTRIBUTING.md ?

**Comment by VSevagen** [comment]
@vchrombie I've update the PR. Could you check it out when you have time ?

File: README.md (hunk 2 above)
Comment (vchrombie) [reply README.md]: I was referring to this https://github.com/chaoss/grimoirelab-tutorial/blob/master/contributing.md#how-to-contribute because it has steps for the old tutorial.

**Comment by vchrombie** [comment]
> @vchrombie I've update the PR. Could you check it out when you have time ?

Thanks @VSevagen, I will review the PR.

File: README.md [hunk 3]
```diff
@@ -1,82 +1,23 @@
 # GrimoireLab Tutorial
 
-[GrimoireLab](http://grimoirelab.github.io/) is free, open source software for software development analytics. It allows you to retrieve data from many kinds of systems with information related to software development, and produce analysis and visualizations with it.
+[GrimoireLab](http://grimoirelab.github.io/) is free, open source software for software
+development analytics. It allows you to retrieve data from many kinds of systems with
+information related to software development, and produce analysis and visualizations with
+it.
 
 ![](assets/wizard-y.svg)
```
Comment (vchrombie) [inline README.md:8]: I think we can slowly update the images, I have created some graphics using existing images.







We need not add them in this PR, but I'm just putting a comment over here.

File: README.md (hunk 2 above)
Comment (vchrombie) [reply README.md]: You can include this in the next PRs.

**Review by vchrombie — APPROVED** [review]
Thanks for kicking off the work @VSevagen!!

LGTM!

**Comment by vchrombie** [comment]
You can now open more PRs changing the theme entirely and replacing the content. Make sure you port the not-relevant content to the their own repositories.
